sdk/python/feast/infra/provider.py

```diff
@@ -127,7 +127,7 @@ def ingest_df(
         pass
 
     def ingest_df_to_offline_store(
-        self, feature_view: FeatureView, df: pd.DataFrame,
+        self, feature_view: FeatureView, df: pyarrow.Table,
```

Comment:
why is this interface changing? can you add that to the PR description?

Reply:
Sure left some comments